### ISSUE TITLE
Symlinks from script-output do not work in 0.17

### ISSUE THREAD

**Issue by L0laapk3** [L0laapk3/FactorioMaps#33]
If you would like this to work again, please redirect your concerns to this factorio thread:

https://forums.factorio.com/viewtopic.php?f=7&t=68499

**Comment by fbartho** [comment]
I was able to bypass this by rewriting how the Symlink command was written:

```python
	def linkDir(src, dest):
		print('os.name: {}'.format(os.name))
		if os.path.exists(dest) == True:
			return

		if os.name == 'nt':
			cmd = ("MKLINK", "/J", os.path.abspath(src), os.path.abspath(dest))
		elif os.name == 'posix':
			cmd = "ln -s '{}' '{}'".format(os.path.abspath(src), os.path.abspath(dest))
		else:
			cmd = ("ln", "-s", os.path.abspath(src), os.path.abspath(dest))
		subprocess.check_call(cmd, stdout=subprocess.DEVNULL, shell=True)
```

Not sure why the else-block's command wasn't working.

**Comment by L0laapk3** [comment]
Thats something else entirely, thats been fixed as of 548816420043ef8ecf66d983286df01d86448f5c. This issue is about a sub folder inside the script-output folder being a symlink to, for example, a different drive.

**Comment by L0laapk3** [comment]
This is a workaround, not a complete solution. But since factorio devs don't seem to be changing their mind about this issue anytime soon this will have to do.
